documentation/principles/index.md

```diff
@@ -2,27 +2,20 @@
 layout: product.njk
 order: 1
 title: Principles
-description: Below are our guiding principles
-excerpt: What guides us
+description: Core Cloud aims to be a high quality, easy to operate, strategic platform.
+excerpt: Core Cloud aims to be a high quality, easy to operate, strategic platform.
 tags:
 - homepage
 eleventyNavigation:
   key: principles
   title: Principles
 ---
-{% from "govuk/components/button/macro.njk" import govukButton %}
-<div class="flex-items">
+
+We have {{ collections["principle"].length }} fundamental principles that we use to robustly evaluate the decisions we make.
+
 {% for item in collections["principle"] %}
   <div>
-    <h2 class="govuk-heading-m govuk-!-font-size-27">{{ item.data.title }}</h2>
-    <p class="govuk-body">{{ item.data.excerpt | markdown("inline") }}</p>
-    {% set html = ['Learn more<span class="no-presentation"> about the ', item.data.title, ' principle</span>'] | join %}
-        {{ govukButton({
-          classes: "x-govuk-button--inverse",
-          html: html,
-          href: item.url,
-          isStartButton: false
-        }) }}
+    <h2 class="govuk-heading-m govuk-!-font-size-27">{{ collections["principle"].indexOf(item) +1 }}. {{ item.data.title }}; {{ item.data.excerpt | markdown("inline") }}</h2>
+    <p class="govuk-body">{{ item.content }}</p>
```

Comment:
Not sure why the link has been removed. Now you can't click through to the principle

Reply:
We can talk about this, I wasn't a fan of 7 pages for 20 lines of text - I felt it was too deep, the whole template needs some work around navigation but I didn't want to get too distracted 😀